Dockerfile-worker-skaffold

```diff
@@ -0,0 +1,26 @@
+# syntax = docker/dockerfile:experimental
+# Build the manager binary
+FROM golang:1.12.5 as builder
+
+WORKDIR /workspace
+
+COPY . .
+
+RUN --mount=type=cache,target=/go/pkg/mod \
+    --mount=type=cache,target=/root/.cache/go-build \
+    CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build \
+    -mod=vendor \
+    -installsuffix netgo \
+    -tags netgo \
+    -o worker \
+    src/server/cmd/worker/main.go
```

Comment:
Same suggestion as above re: `src/.../worker/main.go` vs `src/.../worker` (again, presuming the suggestion even makes sense)